analyzers/src/SonarAnalyzer.CSharp/Rules/UnusedPrivateMember.cs

```diff
@@ -43,6 +43,8 @@ public sealed class UnusedPrivateMember : SonarDiagnosticAnalyzer
         private const string S4487DiagnosticId = "S4487";
         private const string S4487MessageFormat = "Remove this unread {0} field '{1}' or refactor the code to use its value.";
 
+        private const string Private = "private";
```

Comment:
We should extract this to [SyntaxConstants](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/src/SonarAnalyzer.CSharp/Constants/SyntaxConstants.cs) or some other class. This string is hard coded in multiple places in our solution like:
https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/src/SonarAnalyzer.CSharp/Rules/ClassWithOnlyStaticMember.cs#L70
https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/src/SonarAnalyzer.CSharp/Rules/ClassWithOnlyStaticMember.cs#L88
https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/src/SonarAnalyzer.CSharp/Rules/ImplementISerializableCorrectly.cs#L165